gulp-tasks/test.js

```diff
@@ -38,25 +37,25 @@ gulp.task('download-browsers', function() {
   });
 });
 
-gulp.task('mocha', ['build', 'download-browsers'], () => {
-  const mochaOptions = {};
+gulp.task('mocha', ['build'], () => {
+  const mochaOptions = {
+    env: {
+      projectOrStar: global.projectOrStar,
+    },
+  };
+
   if (global.cliOptions.grep) {
     mochaOptions.grep = global.cliOptions.grep;
   }
-  const testGlob = `packages/${global.projectOrStar}/test/*.js`;
-  const testFiles = glob.sync(testGlob);
-  if (testFiles.length === 0) {
-    // Mocha fails when no tests are found so return early.
-    return;
-  }
-  return gulp.src(testGlob, {read: false})
+
+  return gulp.src('test/bootstrap.js', {read: false})
```

Comment:
I'm seeing instances where there are individual errors in tests, but the `gulp test` process exits with a 0 status:

E.g.
https://travis-ci.org/GoogleChrome/sw-helpers/jobs/219368904#L1829 is an error, but the Travis CI build is green because the test exited with a 0: https://travis-ci.org/GoogleChrome/sw-helpers/jobs/219368904#L2466

Can you double-check this modified code and see what might be causing the errors not to bubble up and trigger the `.once('error', ...)` listener?